docs/user/dql/functions.rst

```diff
@@ -1756,6 +1756,28 @@ Examples::
     +-----------------------------------+
 
 
+GET_FORMAT
+----------
+
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a string value containing string format specifiers based on the input arguments
+
+Argument type: TYPE, STRING
```

Comment:
List arg acceptable values somehow.

Reply:
And for second arg too

Reply:
Fixed in 2d33c7cc0c12bc89d567b81ca2f8db4d8ab976ad